### ISSUE TITLE
valid-title no longer reports emptyTitle errors since v1.1.38

### ISSUE THREAD

**Issue by y-hsgw** [vitest-dev/eslint-plugin-vitest#687]
Since upgrading to Vitest v1.1.38, the `valid-title` rule no longer reports emptyTitle errors. It seems to be a false negative, as empty titles are not flagged anymore.
This was working correctly in v1.1.37.

```ts
it('', () => { // Should report emptyTitle error, but doesn't
  expect(1).toBe(1);
});
```

eslint.config.js
 ```js
 {
  plugins: {
    vitest,
  },
  rules: {
    ...vitest.configs.recommended.rules,
  },
  settings: {
    vitest: {
      typecheck: true,
    },
  },
  languageOptions: {
    globals: {
      ...vitest.environments.env.globals,
    },
  },
},
```

Related PR: https://github.com/vitest-dev/eslint-plugin-vitest/pull/682
